README.md

```diff
@@ -1,2 +1,16 @@
-# hw01
-Repository for first CFSS homework assignment
+---
+title: "A Brief Introdution"
+author: "Eleanor Koh"
+date: "11/01/2017"
+output: html_document
+---
+I like elephants, coffee and *Barefoot Contessa*. 
+
+**3 fun facts**
+
+1. Apart from myself, elephants are the only mammals that can't jump. That's why we get along, elephants and I. Elephants are also remarkably sentient: they can laugh, cry and *hug* by wrapping their trunks together. 
+
+2. I own an espresso pot, a french press, and a v60 (this is a funnel) and a jar of instant coffee. This means that I can make coffee *4* different ways. 
```

Comment:
You could also number the list like

```
1. Apart from myself...
1. I own a...
1. You know those...
```

and Markdown will auto-number each item. Like this:

1. Apart from myself...
1. I own a...
1. You know those...

This way if you ever decide to reorder the list items, you don't have to manually change the numbers